File: app/src/main/java/nithra/gift/suggestion/shop/birthday/marriage/ExitScreen.java
package nithra.gift.suggestion.shop.birthday.marriage;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Handler;

public class ExitScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_exit_screen);
        Handler handel = new Handler();
        handel.postDelayed(() -> {
            finish();
        }, 1500);
    }
}

File: app/src/main/java/nithra/gift/suggestion/shop/birthday/marriage/ExitScreen.kt
package nithra.gift.suggestion.shop.birthday.marriage

import android.os.Bundle
import android.os.Handler
import android.view.Window
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity

class ExitScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        this.window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )
        setContentView(R.layout.activity_exit_screen)
        val handel = Handler()
        handel.postDelayed({ this.finish() }, 1500)
    }
}